.drone.yml

```diff
@@ -5126,3 +5126,235 @@ steps:
+
+---
+##############################################################################################################
+# This pipeline runs tezedge and ocaml nodes in isolation, bootsrapped to 20000
+# and performs `wrk` and `wrk2` tests over them to compute throughput and latency
+##############################################################################################################
+kind: pipeline
+name: performance/wrk-tests-009-florence
+# default build goes with: --network "florencenet"
+# integration test is using env variables:
+# - BLOCK_HEAD - top block level used by bootstrap
+# - OCAML_NODE_RPC_CONTEXT_ROOT_1: running ocaml node, e.g.: http://ocaml-node:8732
+# - TEZEDGE_NODE_RPC_CONTEXT_ROOT: running tezedge node, e.g.: http://tezedge-updated-run:18732
+
+clone:
+  disable: true
+
+environment:
+  NETWORK: florencenet
+  BLOCK_HEAD: 20000 # top block to bootstrap to
+  WRK_TEST_DURATION: 10  # in seconds
+  WRK2_TEST_DURATION: 20  # in seconds
+  WRK2_REQ_RATE: 100  # in seconds
+  SODIUM_USE_PKG_CONFIG: 1
+  MAX_LATENCY_THRESHOLD: 10 # in %
+  THROUGHPUTL_THRESHOLD: 10 # in %
+  OCAML_NODE_RPC_CONTEXT_ROOT_1: http://ocaml-node:8732
+  OCAML_NODE_P2P_PEERS: ocaml-node:9734
+  TEZEDGE_NODE_RPC_CONTEXT_ROOT: http://tezedge-updated-run:18732
+  TEZEDGE_NODE_TARGET_BRANCH_RPC_CONTEXT_ROOT: http://tezedge-old-run:18732
+  BUILD_ARTIFACTS_PATH: /artifacts/build_${DRONE_BUILD_NUMBER}
+  TEST_ARTIFACTS_PATH: /tests/build_${DRONE_BUILD_NUMBER}
+  CACHE_DATA_PATH: /data/cache/build_${DRONE_BUILD_NUMBER}/${DRONE_STAGE_NAME}
+
+steps:
+- name: prepare-data
+  image: alpine/git
+  user: root
+  volumes:
+    - name: cache
+      path: /data/cache
+  commands:
+    - rm -rf $${CACHE_DATA_PATH}
+    - mkdir -p $${CACHE_DATA_PATH}
+
+# just run ocaml 3 snapshoted nodes
+- name: ocaml-node
+  user: root
+  image: tezos/tezos:v9-release
+  detach: true
+  volumes:
+    - name: build
+      path: /artifacts
+    - name: data
+      path: /home/tezos/data
+    - name: cache
+      path: /data/cache
+  commands:
+    - ip -o -4 addr show | awk -F '[ /]+' '/global/ {print $4 ":9734"}' > $${CACHE_DATA_PATH}/ocaml_ip.txt
+    - rm -f /home/tezos/data/lock
+    - cp $${BUILD_ARTIFACTS_PATH}/build_files/identities/identity_1.json /home/tezos/data/ocaml-node-$${NETWORK}-snapshot-data-1/identity.json
+    - tezos-node config reset --data-dir /home/tezos/data/ocaml-node-$${NETWORK}-snapshot-data-1 --network $${NETWORK} --no-bootstrap-peers
+    - tezos-node run --data-dir /home/tezos/data/ocaml-node-$${NETWORK}-snapshot-data-1 --history-mode archive --rpc-addr 0.0.0.0:8732 --net-addr 0.0.0.0:9734 --network $${NETWORK} --no-bootstrap-peers
+
+- name: wait-for-ocaml-snapshoted-nodes
+  image: simplestakingcom/tezos-node-bootstrap:latest
+  pull: if-not-exists
+  commands:
+    - tezos-node-bootstrap bootstrap --level=$${BLOCK_HEAD} --nodes $${OCAML_NODE_RPC_CONTEXT_ROOT_1} $${OCAML_NODE_RPC_CONTEXT_ROOT_2} $${OCAML_NODE_RPC_CONTEXT_ROOT_3}
```

Comment:
@akoptelov 
please, remove `${OCAML_NODE_RPC_CONTEXT_ROOT_2} $${OCAML_NODE_RPC_CONTEXT_ROOT_3}`  they are not used

Reply:
Argh...